speechtotextv1/speech_to_text_v1.go

```diff
@@ -4985,6 +4985,16 @@ type RecognizeOptions struct {
 
 	// Allows users to set headers to be GDPR compliant
 	Headers map[string]string
+
+	// Websocket only. The action that is to be performed. Allowable values: start, stop
```

Comment:
Instead of adding all these params here, we can put it in a separate struct `RecognizeUsingWebsocketsOptions`. 

Also move the struct to `speech_to_text_v1_adapter.go`. We automatically generate the SDKS, and this would be a handwritten change, so we can move it to the adapter.